src/App.jsx

```diff
@@ -14,70 +16,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  function setHeading(selectedGood) {
+    return goods.includes(selectedGood)
+      ? `${selectedGood} is selected`
+      : 'No goods selected';
+  }
```

Comment:
Good trick🔥 but move this helper function out of your component function. Why? Because on each render of App component new `setHeading function` will be created in memory.